#Request - peticion que se hace al servidor mediante el servicio
string object_name
---
#Response - datos que devuleve el servidor al atender la petici√≥n
float64 object_distance
bool success
string status_message